### PR TITLE
Progress on benchmarking package and FACE project related scripts

### PR DESCRIPTION
**Pull request by bcow**
A portion of this pull request contains duplicate commits that were created by a mistake I made using github - apologies for mucking up the commit history - I thought this should wait until after the new build.

### CODE REVIEW

File: utils/R/remote.R [hunk 1]
```diff
@@ -201,7 +201,14 @@ remote.execute.R <- function(script, host="localhost", user=NA, verbose=FALSE, R
   } else {
     remote <- ifelse(is.na(user), host, paste(user, host, sep='@'))[[1]]
     result = system2('ssh', c('-T', remote, R, "--vanilla"), stdout=verbose, stderr=verbose, input=input)
-    remote.copy.file(host, tmpfile, user, "localhost", tmpfile)
+    
+    # Tunnel is hardcoded in for geo ... where should this be specified?
+    host_list <- list(name=host, user=user, tunnel="/tmp/geo.tunnel")
+    remote.copy.from(host_list, tmpfile, tmpfile, delete=TRUE)
+
```
Comment (mdietze) [inline utils/R/remote.R]: @robkooper could you check what @bcow is doing here, I'm unfamiliar with how you have tunnels set up.

Comment (robkooper) [inline utils/R/remote.R]: The code should be modified to pass in the host object to the remote.R function as is done with the other functions. This way it has the username,  host, tunnel, etc.

**Comment by robkooper** [comment]
Can we hold off on this pull request until after 1.4.5 is released officially, unless you want it to be part of the 1.4.5 release.

**Comment by mdietze** [comment]
Rob, we've got a few more things still in the pipeline for 1.4.5 that I really DO want to get in, so I think it's fine to bring this in if we're going to also bring in the global met and preles still. I don't think it's going to break anything

**Pull request closed without merging**